cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -64,9 +65,21 @@ spec:
           - name: grafana-persistent-storage
             mountPath: /var
       volumes:
-      - name: influxdb-persistent-storage
-        persistentVolumeClaim:
-          claimName: influxdb-claim
       - name: grafana-persistent-storage
         emptyDir: {}
-  serviceName: monitoring-influxdb
+  volumeClaimTemplates:
+  - metadata:
+      name: influxdb-persistent-storage
+      namespace: kube-system
+      #labels:
+        #kubernetes.io/cluster-service: "true"
+      annotations:
+        volume.alpha.kubernetes.io/storage-class: "anything"
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 10Gi
+      volumeName: influxdb-pv
```

Comment:
why do you need this? either use storage-class, or pre-provision your pv.
If you pre-provision your pv, what is the expected behavior when you scale the petset?